app/src/main/java/com/google/samples/apps/sunflower/compose/home/HomeScreen.kt

```diff
@@ -85,11 +85,11 @@ fun HomeScreen(
                 scrollBehavior = scrollBehavior
             )
         }
-    ) {
+    ) { contentPadding ->
         HomePagerScreen(
             onPlantClick = onPlantClick,
             pagerState = pagerState,
-            modifier = Modifier.padding(it)
+            Modifier.padding(top = contentPadding.calculateTopPadding())
```

Comment:
Did you add padding to the bottom of the screen too so it can scroll the content out from underneath the bar?

Reply:
There's already padding applied so it offsets correctly when you scroll all the way down in the plant list screen: https://github.com/android/sunflower/blob/main/app/src/main/java/com/google/samples/apps/sunflower/compose/plantlist/PlantListScreen.kt#L58

...though technically it doesn't take into account `navigationBarsPadding` so let me add that.